includes/data-stores/class-wc-product-data-store-cpt.php

```diff
@@ -680,7 +680,7 @@ protected function update_version_and_type( &$product ) {
 	 */
 	protected function clear_caches( &$product ) {
 		wc_delete_product_transients( $product->get_id() );
-		wp_cache_delete( 'product-' . $product->get_id(), 'products' );
+		wp_cache_delete( 'object-' . $product->get_id(), 'products' );
```

Comment:
Is this correct? All of the product caching will still happen under [`product-`](https://github.com/woocommerce/woocommerce/blob/bf384d07be0c5622b0f81e14d4e006ba10500347/includes/class-wc-product-factory.php#L51), so this won't actually clear the cache currently. It looks to me like all of the product caching still needs to be converted over to `object-`.